content/articles/coming-soon-landing-page-using-javascript-timers/index.md

```diff
@@ -0,0 +1,434 @@
+### Introduction
+A coming soon landing page is a temporary home page that informs visitors that your website is in the process of being built or will be available soon.   
+Since it lacks a header or a footer, visitors can not navigate to the rest of your website, hence it is important to capture what is under development to give as much information as possible.
+
+Every coming soon page includes a statement encouraging visitors to return at a later time, as well as any other pertinent information concerning the website that is under development.
+
+This article will teach you how to make a coming soon landing page, but first things first, you will learn about JavaScript timers to get you started.
+
+#### Table Of Contents
+- [Introduction](#introduction)
+- [JavaScript timers overview](#javascript-timers-overview)
+- [code execution after a delay](#code-execution-after-a-delay)
+- [Code execution at regular intervals](#code-execution-at-regular-intervals)
+- [Cancelling a timer](#cancelling-a-timer)
+- [Template section](#template-section)
+- [Best practices](#best-practices)
+- [Conclusion](#conclusion)
+
+### JavaScript timers overview
+A timer is a feature that allows us to run a program at a predetermined time. Timers can be used to postpone code execution so that it doesn't finish at the same time as an event or a page loading. For example, timers can be used to alter your website's advertisement banners at regular intervals or to display a real-time clock, among other things.  
+
+In JavaScript, there are two timer functions: `setTimeout ()` and `setInterval ()`. Browsers allow the implementation of timer functions, and their implementations differ from one browser to another.
+
+In browsers, the window interface is the parent to main timer features. Since the window interface makes its elements available globally in the main JavaScript scope, `setTimeout()` may be used in the browser console.
+
+### Code execution after a delay
+The `setTimeout()` function is used to only run a function or a piece of code once after a given amount of time has passed.
+
+**syntax**
+
+```JavaScript

```

Comment:
```javascript``` . Not ```JavaScript```. Remember these are not rendered on the screen.

Reply:
make changes where applicable.

Reply:
Thanks, for the corrections I have rectified them